platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu-db-util.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Pensando DPU health status provider and chassis state db util service
```

Comment:
@prgeor , can you check why this is needed per dpu vendor? this should be generic?

Reply:
@lguohan ack

Reply:
There is dpu-state as part of PMON-HLD required for cisco dpu monitoring from NPU side. We have sysmon running inside dpu pensando firmware which monitors dpu health and publish it via grpc, we need grpc listener continuously listening from DPU sonic host side, hence this service/daemon is required. This cannot be added as part of healthd. Healthd have user defined checker as a hook for platform specific code but it invokes periodically which something not needed here.

Reply:
Other modules running inside dpu-db-utild have been commented out as per other comments as per phase-I

Reply:
@shanshri usecase of dpu-db util is not clear. can we align with common design? I did not understand why we cannot align with healthd. Polling is fine.

Reply:
@prgeor, we have two modules in it i.e event_listener and grpc server for fetching reboot_cause for NPU as per latest design. Both uses grpc and cannot be achieved using polling. We need to come up with some sort of pmon daemon/service which will befit this requirement if not already present in system. This will require lot of effort and testing. We are planning to do it as part of Phase-II